crates/neon-runtime/src/napi/bigint.rs

```diff
@@ -0,0 +1,57 @@
+use crate::napi::bindings as napi;
+use crate::raw::{Env, Local};
+use std::mem::MaybeUninit;
+
+/// Create a new bigint object
+///
+/// # Safety
+///
+/// `env` is a raw pointer. Please ensure it points to a napi_env that is valid for the current context.
+pub unsafe fn new_bigint(env: Env, value: i64) -> Local {
+    let mut local = MaybeUninit::zeroed();
+    let status = napi::create_bigint_int64(env, value, local.as_mut_ptr());
+    assert_eq!(status, napi::Status::Ok);
+    local.assume_init()
+}
+
+/// Create a new bigint object
+///
+/// # Safety
+///
+/// `env` is a raw pointer. Please ensure it points to a napi_env that is valid for the current context.
+pub unsafe fn new_bigint_from_u64(env: Env, value: u64) -> Local {
+    let mut local = MaybeUninit::zeroed();
+    let status = napi::create_bigint_uint64(env, value, local.as_mut_ptr());
+    assert_eq!(status, napi::Status::Ok);
+    local.assume_init()
+}
+
+/// Get the value of a bigint object
+///
+/// # Safety
+///
+/// `env` is a raw pointer. Please ensure it points to a napi_env that is valid for the current context.
+/// `Local` must be an NAPI value associated with the given `Env`
+pub unsafe fn value_i64(env: Env, p: Local) -> (i64, bool) {
```

Comment:
Can you return a `Result` instead? It feels like more idiomatic Rust to me.

```suggestion
pub unsafe fn value_i64(env: Env, p: Local) -> Result<i64, i64> {
```

`Result::Err` indicates that loss occurred.

Reply:
Great idea 👍 
I struggled with what to return here.